### PR TITLE
implement function let Koalas DataFrame constuctor take Koalas Series

### PR DESCRIPTION
**Pull request by yuwillrun**
Koalas DataFrame constuctor should take Koalas Series https://github.com/databricks/koalas/issues/466

### CODE REVIEW

File: databricks/koalas/tests/test_dataframe.py [hunk 1]
```diff
@@ -69,6 +69,12 @@ def test_dataframe(self):
 
         self.assertEqual(ddf.a.notnull().alias("x").name, "x")
 
+        # check ks.DataFrame(ks.Series)
+        kser = pd.Series([1, 2, 3], name='x')
+        pser = ks.Series([1, 2, 3], name='x')
+        print(kser, pser)
```
Comment (HyukjinKwon) [inline databricks/koalas/tests/test_dataframe.py]: Can we remove this print?

File: databricks/koalas/tests/test_dataframe.py [hunk 2]
```diff
@@ -69,6 +69,12 @@ def test_dataframe(self):
 
         self.assertEqual(ddf.a.notnull().alias("x").name, "x")
 
+        # check ks.DataFrame(ks.Series)
+        kser = pd.Series([1, 2, 3], name='x')
+        pser = ks.Series([1, 2, 3], name='x')
```
Comment (ueshin) [inline databricks/koalas/tests/test_dataframe.py]: `kser` and `pser` are upside down?

Comment (yuwillrun) [reply databricks/koalas/tests/test_dataframe.py]: I'll fix it right now.Excuse me. How to mention it in the doc?

File: databricks/koalas/frame.py [hunk 3]
```diff
@@ -245,6 +245,13 @@ def __init__(self, data=None, index=None, columns=None, dtype=None, copy=False):
             else:
                 super(DataFrame, self).__init__(_InternalFrame(
                     data, data_columns=index.data_columns, index_map=index.index_map))
+        elif isinstance(data, ks.Series):
```
Comment (HyukjinKwon) [inline databricks/koalas/frame.py]: I think he meant the docstring here. yea we should fix

File: databricks/koalas/tests/test_dataframe.py (hunk 2 above)
Comment (ueshin) [reply databricks/koalas/tests/test_dataframe.py]: Maybe we should say like `If data is a Koalas Series, ...`.

File: databricks/koalas/frame.py (hunk 3 above)
Comment (ueshin) [reply databricks/koalas/frame.py]: yup, I meant the docstring.

Comment (yuwillrun) [reply databricks/koalas/frame.py]: > I think he meant the docstring in this constructor. yea we should fix

Sorry, im not familiar with it. Please can you fix the docstring?

Comment (ueshin) [reply databricks/koalas/frame.py]: Sure, I'll add it later.

**Comment by ueshin** [comment]
Thanks! merging.

**Pull request merged**